views/template/documentation.pug

```diff
@@ -0,0 +1,8 @@
+extends layout.pug
+
+block head
+	center
```

Comment:
do not use center tags. You are also missing the navigation on this page. You need a navigation block here not a head block?